cache/lfu_test.go

```diff
@@ -0,0 +1,178 @@
+package cache
```

Comment:
For testing, I really recommend you use table testing rather than multiple tests such as `TestLFU1` etc.

Also, really think you should use black box testing in this case. You may have missed something trivial that is not discoverable locally. Use
```suggestion
package cache_test
```